ops/log.py

```diff
@@ -26,7 +26,11 @@ def __init__(self, model_backend, level=logging.DEBUG):
         self.model_backend = model_backend
 
     def emit(self, record):
-        """Send the specified logging record to the Juju backend."""
+        """Send the specified logging record to the Juju backend.
+
+        This method is not used directly by the Operator Framework code, but by
+        logging.Handler itself as part of the logging machinery.
```

Comment:
this would benefit from a bit of markup,
```suggestion
        :class:`logging.Handler` itself as part of the logging machinery.
```